Instagram Follow- NotFollow/main.py

```diff
@@ -0,0 +1,92 @@
+from selenium import webdriver
```

Comment:
Print the list of followers and non-followers in a more attractive way. Maybe an ascii table with two columns: followers | non followers